internal/db/rdb/mysql/mysql_test.go

```diff
@@ -2793,10 +2793,10 @@ func Test_mySQLClient_SetVectors(t *testing.T) {
 	}
 }
 
-func Test_mySQLClient_deleteVector(t *testing.T) {
+func Test_mySQLClient_DeleteVector(t *testing.T) {
```

Comment:
Since it is test function, it is hard to avoid complex implementation.
I will ignore this warning.
Please let me know it you have any other comment for this warning. :)